src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -17353,3 +17353,236 @@ def test_aks_update_with_disable_upstream_kubescheduler_user_configuration(
                 self.is_empty(),
             ],
         )
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="westus2"
+    )
+    def test_aks_create_with_blue_green_upgrade_nodepool(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=136612802&view=results) to validate the change.